index.d.ts

```diff
@@ -0,0 +1,60 @@
+export interface Action {
+  type: string;
```

Comment:
Is this correct? Do we expect action type to always be `string`, or should it be `any`?